webpack.coverage.js

```diff
@@ -0,0 +1,33 @@
+const config = require('./webpack.dev');
```

Comment:
Is there a reason we can't put this back into the `webpack.dev` config? I'm worried about managing another config

Reply:
I think it's a good idea not to make the dev build last longer when we don't need coverage during development (but I haven't measured the difference yet). We could also stick this inside karma conf just like the old config has istanbul-instrumentation-loader, but I think this is cleaner.